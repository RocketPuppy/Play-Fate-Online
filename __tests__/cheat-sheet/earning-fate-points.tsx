import * as React from "react";
import * as renderer from "react-test-renderer";
import EarningFatePoints from "../../src/cheat-sheet/earning-fate-points";

test("renders all the pieces", () => {
  const render = renderer.create(<EarningFatePoints />);

  expect(render.toJSON()).toMatchSnapshot();
});
