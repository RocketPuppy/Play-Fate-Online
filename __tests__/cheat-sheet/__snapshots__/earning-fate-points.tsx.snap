// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders all the pieces 1`] = `
<div>
  <h2>
    Earning Fate Points
  </h2>
  <p>
    Earn fate points when you:
  </p>
  <ul>
    <li>
      Accept a compel
    </li>
    <li>
      Have your aspects invoked against you
    </li>
    <li>
      Concede a conflict
    </li>
  </ul>
</div>
`;
